### ISSUE TITLE
Need a test case that seeds comparisons

### ISSUE THREAD

**Issue by khatchad** [ponder-lab/Null-to-Optional-Eclipse-Plugin#98]
## In
```java
Object o = new Object();
boolean b = o == null;
```
## Out
```java
Optional<Object> o = Optional.ofNullable(new Object());
boolean b = o.isPresent();
```

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/274838528047159/839796804077357)

**Comment by khatchad** [comment]
@walleXD Do you think you could handle this one?

**Comment by walleXD** [comment]
@khatchad I can handle it

**Comment by walleXD** [comment]
@khatchad & @orenwf Moving forward, are we avoiding `Optional.empty()` & instead using `Optional.ofNullable(new Object())`?

**Comment by walleXD** [comment]
Fix in #102
